docs/backend/db/analysis.md

```diff
@@ -0,0 +1,48 @@
+# DB Counts
```

Comment:
@Sigfried  the output: https://github.com/jhu-bids/TermHub/blob/db-counts-update/docs/backend/db/analysis.md